src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -17,27 +17,33 @@
 
 // This is a generated sample ("FlattenedAsync", "empty_response_type_with_response_handling")
 
-// [START sample]
-// FIXME: import everything this sample needs
-public class DeleteShelfAsyncFlattenedAsyncEmptyResponseTypeWithResponseHandling
+namespace Google.Example.Library.V1.Samples
 {
-    // [START sample_core]
-    /// <summary>
-    /// Test response handling for methods that return empty
-    /// </summary>
-    public static async Task SampleDeleteShelfAsync()
-    {
-        LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-        ShelfName name = new ShelfName("[SHELF_ID]");
-        await libraryServiceClient.DeleteShelfAsync(name);
-        // FIXME: inspect the results
-    }
-    // [END sample_core]
+    // [START sample]
+    using Google.Example.Library;
+    using apis = Google.Example.Library.V1;
```

Comment:
Where did you copy this from?

[Should be](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/8665b94b10f8ad99b09004a180f21209688a05f9/speech/api/Recognize/Recognize.cs#L19):
```cs
using Google.Example.Library.V1;
```